docs/advanced-usage/satellite-domains.mdx

```diff
@@ -51,6 +51,13 @@ To access authentication state from a satellite domain, users will be transparen
   - In production, the satellite domain will be `satellite.dev`.
   - In development, the satellite domain will be `localhost:3001`.
 
+  > [!IMPORTANT]
+  > To use a satellite domain in production, add a **CNAME record** for your satellite subdomain in your DNS provider's settings, pointing it to your deployed app's domain. For example, if your satellite domain is `satellite.dev`, and your deployed app is hosted at `your-app.com`, you would create a CNAME record for `satellite.dev` pointing to `your-app.com`.
```

Comment:
This isn't how CNAME records are specified - could we make this a bit more specific? Like, what is the text of the record?